client/src/App/App.tsx

```diff
@@ -54,42 +64,44 @@ const App: React.FC = () => {
             {/* Home Page view */}
             <ConnectedPrivateRoute
               redirectPath={APP_CALLBACK_URL}
-              disableLoginProtection={false}
+              disableLoginProtection={DISABLE_LOGIN_PROTECTION}
               exact
               path="/"
               component={ConnectedHomeComponent}
             />
             <ConnectedPrivateRoute
               redirectPath={APP_CALLBACK_URL}
-              disableLoginProtection={false}
+              disableLoginProtection={DISABLE_LOGIN_PROTECTION}
               exact
-              path="/admin"
-              component={ConnectedAdminView}
+              path={URL_ADMIN}
+              component={(props: any) => (
+                <ConnectedUserList {...props} keycloakBaseURL={KEYCLOAK_API_BASE_URL} />
+              )}
             />
             <ConnectedPrivateRoute
               redirectPath={APP_CALLBACK_URL}
-              disableLoginProtection={false}
+              disableLoginProtection={DISABLE_LOGIN_PROTECTION}
               exact
-              path="/user/edit/:userId"
-              render={(props: any) => (
-                <ConnectedUserCredentials {...props} keycloakBaseURL={KEYCLOAK_API_BASE_URL} />
+              path={`${URL_USER_EDIT}/:${ROUTE_PARAM_USER_ID}`}
+              component={(props: any) => (
```

Comment:
we have any here, intentional? if I remember correctly this type should be sth along `RouteComponentProps<RouteParamProps>`